dev_guide/projects.adoc

```diff
@@ -194,3 +194,160 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+ifdef::openshift-online[]
+[[project-collaboration-in-online-pro]]
+== Project Collaboration in {product-title} Pro
+
+Every {product-title} Pro account has the ability to add up to 50 collaborator
+users per subscription. These collaborator users are granted cluster access from
+{product-title} Pro account subscribers so that they can collaborate on projects
+hosted with {product-title}. This allows multiple users to gain access to
+projects under a single subscription, without having to pay a monthly fee for
+every account.
+
+[NOTE]
+====
+Existing {product-title} Pro users do not need to be added as collaborators.
+====
+
+Collaboration is not available in {product-title} Starter.
+
+[[collaboration-restrictions]]
+=== Collaboration Restrictions
+
+Collaborators can only access the resources within the projects that they have
+been granted access to. Also, their ability to view, edit, and manage the
+project resources will depend on the specific role that they have been granted
+within the project.
+
+[[project-collaboration-adding-collaborators]]
+=== Adding Collaborators
+{product-title} Pro subscribers can add collaborators by following these steps:
+
+. Each user you want to add as a collaborator must create a free account at
+ link:https://developers.redhat.com/[developers.redhat.com]. Once your
+ collaborator has confirmed their Red Hat Developers account, you can add them
+ to your subscription.
```

Comment:
How does a customer do this? How can Support do this?

Reply:
The customer simply needs to have their collaborator create a Red Hat Developers account.  Going through the normal registration process and email verification.  Once they have a developer account, they'll be able to see their login ID (see below), which is what is needed to add them to the pro user's collaboration management page.